### ISSUE TITLE
Incorrect statement about ADA (Americans with Disabilities Act)

### ISSUE THREAD

**Issue by maya** [18F/web-design-standards#884]
The homepage has this stated:

>Accessibility out of the box

>These standards were built with a priority on 508 compliance and ADA accessibility at every step of the design process. From colors to code, everything you need to meet high standards of accessibility are baked into these tools.

However, the accessibility work has nothing to do with Americans with Disabilities Act (ADA), just Section 508 of the Rehabilitation Act, so it should be removed.

Also a note from @rusty1992 stated: Should be Section 508 conformance not compliance. Agencies comply with the law when procuring products and services and industry and development conforms with the requirements.

**Comment by awfrancisco** [comment]
I like the second one.

**Comment by maya** [comment]
@awfrancisco :+1: :zap:
